src/main/resources/db/changelog/changes/06-add-isDeleted-column-to-users-table.yaml

```
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-is-deleted-to-users
+      author: dmytro

```

```suggestion
      author: dmytro_potapov
```
it's better to provide full name as author of changeSet